app/src/main/javascript/bundles/experiment-page/html/index.html

```diff
@@ -326,7 +326,7 @@
     ];
 
     experimentPageDemo.render({
-        atlasUrl: 'https://wwwdev.ebi.ac.uk/gxa/sc/',
+        atlasUrl: 'http://localhost:8080/gxa/sc/',
```

Comment:
Maybe this needs to be  updated to `wwwdev`

Reply:
Actually it doesn't matter, I would like to test the localhost actually. And normally we should test with local not the production environment.